config/Reader.alpha.xcconfig

```diff
@@ -3,4 +3,4 @@
 
 WP_PUSH_NOTIFICATION_APP_ID = TBD
 WP_BUILD_CONFIGURATION = alpha
-WP_APP_URL_SCHEME = TBD
+WP_APP_URL_SCHEME = wpreader
```

Comment:
This value could also be specialized by build configuration, e.g.:

https://github.com/wordpress-mobile/WordPress-iOS/blob/2281871ee128e48c7e6d1781cba16a041d213c5a/config/WordPress.debug.xcconfig#L6

https://github.com/wordpress-mobile/WordPress-iOS/blob/2281871ee128e48c7e6d1781cba16a041d213c5a/config/WordPress.release.xcconfig#L6

but I'm not sure if we need to?

Reply:
`WP_APP_URL_SCHEME` should be one for each app identifier. If I'm not mistaken, debug and release builds share the same app identifier, but alpha builds use a different one.